site/source/docs/api_reference/val.h.rst

```diff
@@ -262,6 +262,43 @@ Guide material for this class can be found in :ref:`embind-val-guide`.
 
       .. note:: This method requires :ref:`Asyncify` to be enabled.
 
+  .. cpp:function:: val operator co_await() const
+
+   The ``co_await`` operator allows to await JavaScript promises represented by ``val``.
```

Comment:
Change to: The ``co_await`` operator allows awaiting on JavaScript promises represented by ``val``.

Reply:
Small ask if you don't mind: could you please use Github's "Add a suggestion" block for small suggestions like these? Then I could quickly accept them directly from UI.

Reply:
I'll do it myself for these few.

```suggestion
   The ``co_await`` operator allows awaiting JavaScript promises represented by ``val``.
```